notebooks/using-mlflow/no-code-deployment/deploying_with_mlflow.ipynb

```diff
@@ -0,0 +1,469 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0c9467c5",
+   "metadata": {},
+   "source": [
+    "# Deploying models with MLflow\n",
+    "\n",
+    "Azure ML supports no-code deployment of models created and logged with MLflow. This means that you don't have to provide a scoring script or an environment. Those models can be deployed to ACI (Azure Container Instances), AKS (Azure Kubernetes Services) or our managed inference services (usually referred as MIR). \n",
+    "\n",
+    "For no-code-deployment, Azure Machine Learning\n",
+    "\n",
+    "- Dynamically installs Python packages provided in the conda.yaml file, this means the dependencies are installed during container runtime.\n",
+    "- The base container image/curated environment used for dynamic installation is `mcr.microsoft.com/azureml/mlflow-ubuntu18.04-py37-cpu-inference` or `AzureML-mlflow-ubuntu18.04-py37-cpu-inference`.\n",
+    "\n",
+    "The following table shows the target support for MLflow models in Azure ML:\n",
+    "\n",
+    "\n",
+    "| Feature | ACI | AKS | MIR |\n",
+    "| :- | :-: | :-: | :-: |\n",
+    "| Deploying models logged with MLflow to real time inference | ☑️* | ☑️* | ☑️* |\n",
+    "| Deploying models logged with MLflow to batch inference | ☐** | ☐** | ☑️ |\n",
+    "| Deploying models logged with MLflow locally | ☑️ | ☑️ | ☐*** |\n",
```

Comment:
Does this mean I have a model on my local machine and I want to deploy it to MIR ? That should be possible right ? Please correct me if I am missing something.